src/coordination/coordinator_state_machine.cpp

```diff
@@ -230,9 +362,19 @@ auto CoordinatorStateMachine::create_snapshot_internal(ptr<snapshot> snapshot) -
               fmt::format("Create snapshot internal, last_log_idx={}", snapshot->get_last_log_idx()));
 
   auto ctx = cs_new<SnapshotCtx>(snapshot, cluster_state_);
+  if (durability_store_) {
+    MG_ASSERT(durability_store_->Put(std::string{kSnapshotIdPrefix} + std::to_string(snapshot->get_last_log_idx()),
```

Comment:
Not sure for assert, I wouldn't crash Memgraph for this.... Log but don't crash IMO